uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/general-settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Prefix this file with `leave-absense-admin-*`. This way we know which scenarios are related to L&A